DigitalExcellence/dex-backend
# added support for created and updated fields

## Description

A few sentences describing the overall goals of the pull request's commits.

## Type of change


## Checklist


## Steps to Test or Reproduce

create a project
now the date should display the current date.
change something of the project.
now the updated field should be updated.

## Link to issue

[#217](https://github.com/DigitalExcellence/dex-frontend/issues/217)
